app.py

```diff
@@ -28,7 +30,17 @@ def create_app():
 
     handler = logging.StreamHandler(sys.stdout)
     app.logger.addHandler(handler)
-    app.logger.setLevel(logging.DEBUG)
+    app.logger.setLevel(os.getenv('LOG_LEVEL', 'DEBUG'))
```

Comment:
Minor, but worth noting that you've added a (sensible!) warning not to set to DEBUG for prod, but then you've set the default if nothing is set to DEBUG - INFO might be a more sensible default